content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -212,4 +215,14 @@ and `requests` are set equal to `limits` when not explicitly specified. And the
 container's resource limit for the CPU resource is an integer greater than or
 equal to one. The `nginx` container is granted 2 exclusive CPUs.
 
-
+#### Static policy options
+
+If the `full-pcpus-only` policy option is specified, the static policy will always allocate full physical cores.
+You can enable this option using adding `full-pcups-only=true` to the CPUManager policy options.
+By default, without this option, the static policy allocates CPUs using a topology-aware best-fit allocation.
+On SMT enabled systems, the policy can allocate individual virtual cores, which correspond to hardware threads.
+This can lead to different containers sharing the same physical cores; this behaviour in turn contributes
+to the [noisy neighbours problem](https://en.wikipedia.org/wiki/Cloud_computing_issues#Performance_interference_and_noisy_neighbors).
+With the option enabled, the policy will verify the CPU requirements of all the container of every pod can
+be fullfilled allocating full physical cores. If this is the case, the pod will be admitted and this
+facet of the noisy neighbours problem is this avoided; otherwise, the pod will be rejected with `SMTAlignmentError`.
```

Comment:
> If this is the case, the pod will be admitted and this
> facet of the noisy neighbours problem is this avoided;

Not good English, please revise.

> SMTAlignmentError

Is this a user visible message or an implementation detail?

Reply:
sure, rephrasing. The error is user-visible.